core/src/main/java/de/danoeh/antennapod/core/storage/FeedSearcher.java

```diff
@@ -25,19 +25,20 @@
      */
     public static List<SearchResult> performSearch(final Context context,
                                                    final String query, final long selectedFeed) {
-        final int values[] = {0, 0, 1, 2};
-        final String[] subtitles = {context.getString(R.string.found_in_shownotes_label),
-                context.getString(R.string.found_in_shownotes_label),
+        final int values[] = {2, 1, 0, 0};
```

Comment:
Nope, your explanation makes sense.
Could you maybe add 1-2 sentences to the javadoc? Something like "searches for the query in ..., ... and then ...   and that a component is reported for its first occurence... something like that? (It is fine, though, if you have problems phrasing the behavior or don't think it is actually useful to document it)